include/chainblocks.hpp

```diff
@@ -14,7 +14,7 @@
 #include <vector>
 
 namespace chainblocks {
-constexpr uint32_t CoreCC = 'sink'; // 1936289387
+constexpr uint32_t CoreCC = 'frag'; // 1718772071
```

Comment:
Should I also put the numerical value as a comment in places where I have defined a new enum? That seems like a good idea to detect potential collisions.

In which case, should we have a pattern for that comment? Like `// CC = 1718772071`. That way we can gather all of them with a simple search query.

Reply:
Sounds good, let's also maybe add the 0x hex too, or rather let's use hex instead